package.json

```diff
@@ -37,13 +37,13 @@
 		"version": "oclif-dev readme && git add README.md"
 	},
 	"dependencies": {
-		"@adobe-apimesh/mesh-builder": "1.6.0",
+		"@adobe-apimesh/mesh-builder": "1.6.2-alpha.3",
 		"@adobe/aio-cli-lib-console": "^5.0.0",
 		"@adobe/aio-lib-core-config": "^5.0.0",
 		"@adobe/aio-lib-core-logging": "^3.0.0",
 		"@adobe/aio-lib-env": "^3.0.0",
 		"@adobe/aio-lib-ims": "^7.0.1",
-		"@adobe/plugin-hooks": "0.3.0",
+		"@adobe/plugin-hooks": "0.3.2-beta",
 		"@adobe/plugin-on-fetch": "0.1.0",
```

Comment:
@kmaschi The latest package version for plugin-on-fetch shows `0.1.1-alpha` there in that [repo](https://github.com/adobe/plugin-on-fetch/pull/8/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R3)
So wondering if the on-fetch plugin needs to be released as a full version and reference  that here ?

Reply:
I've updated this version to match what is currently in the alpha mesh builder. Speaking with the team it sounds like those changes are still being tested. The intent of this beta branch is to verify mesh-builder functionality and at least the plugin-hooks changes.